website/docs/tutorial/1c-setting-up-databricks.md

```diff
@@ -5,10 +5,244 @@ id: setting-up-databricks
 
 This guide is under construction
 
-## Get started with Databricks
+## Prerequeistics 
+
+Before starting this tutorial, you will need the following:
+
+- Existing Cloud Provider account (AWS, GCP, Azure)
+- Permissions to create S3 bucket in said account
+
+## Setting up Databricks 
+
+1. For this tutorial, you are going to need a Databricks account. You can sign up by clicking [Try Databricks](https://databricks.com/) on the Databricks homepage. Fill out the form with your user information. 
+
+    <Lightbox src="/img/databricks_tutorial/images/signup_form.png" title="Sign up for Databricks" />
```

Comment:
@coapacetic what is the fix for this?

Reply:
I feel trying and I can't figure out how to get this image bleed haha

Reply:
@JKarlavige do you know the quick fix for cleaning up images bleeding into the right hand side of the page?

@patkearns10 you might know the solution here too!

Reply:
@coapacetic I'm thinking a css adjustment may fix this. Do you have the url for the preview of this page? I can't seem to find it within the preview.

Reply:
@coapacetic added a css commit and the images should now be contained!

Reply:
@JKarlavige we were running into the same issue on a different page, was this a global fix? https://github.com/dbt-labs/docs.getdbt.com/pull/1062#discussion_r802137469

Reply:
@patkearns10 The fix added here should target any images that are clickable and open in a lightbox. So this will fix this issue globally for all lightbox images. If there are other non-lightbox images experiencing a similar issue elsewhere I can add a separate fix for that.